recipes/python-toolchain/bld.bat

```diff
@@ -0,0 +1,16 @@
+setlocal EnableDelayedExpansion
+
```

Comment:
I believe the `FOR` loop below failed without this so I had added it. Though I'm hardly a Windows expert so if you don't think it is needed we can try removing it.